spec/models/case/overturned_ico/sar_spec.rb

```diff
@@ -82,14 +73,13 @@
                       'email'                   => 'stephen@stephenrichards.eu',
                       'ico_officer_name'        => 'Dan Dare'
               }).to_unsafe_hash
+          params[:creator] = creator
```

Comment:
This isn't needed IMHO - actually we have already created, so the validity test isn't very useful. build() is the idiomatic pattern here